spec/features_spec.rb

```diff
@@ -34,12 +43,143 @@
+    [han, luke].each do |record|
+      expect { record.reload }.to(
+        change { record.last_name }.to("(redacted)") &
+        preserve { record.first_name } &
+        preserve { record.email }
+      )
+    end
+  end
+
+  example "Specifying multiple attributes in an attr_masker declaration" do
+    User.class_eval do
+      attr_masker :first_name, :last_name
+    end
+
+    expect { run_rake_task }.not_to(change { User.count })
+
+    [han, luke].each do |record|
+      expect { record.reload }.to(
+        change { record.first_name }.to("(redacted)") &
+        change { record.last_name }.to("(redacted)") &
+        preserve { record.email }
+      )
+    end
+  end
+
+  example "Skipping some records when a symbol is passed to :if option" do
+    User.class_eval do
+      attr_masker :first_name, :last_name, if: :jedi?
+    end
+
+    expect { run_rake_task }.not_to(change { User.count })
+
+    expect { han.reload }.to(
+      preserve { han.first_name } &
+      preserve { han.last_name } &
+      preserve { han.email }
+    )
+
+    expect { luke.reload }.to(
+      change { luke.first_name }.to("(redacted)") &
+      change { luke.last_name }.to("(redacted)") &
+      preserve { luke.email }
+    )
+  end
+
+  example "Skipping some records when a lambda is passed to :if option" do
+    User.class_eval do
+      attr_masker :first_name, :last_name, if: ->(r){ r.jedi? }
+    end
+
+    expect { run_rake_task }.not_to(change { User.count })
+
+    expect { han.reload }.to(
+      preserve { han.first_name } &
+      preserve { han.last_name } &
+      preserve { han.email }
+    )
+
+    expect { luke.reload }.to(
+      change { luke.first_name }.to("(redacted)") &
+      change { luke.last_name }.to("(redacted)") &
+      preserve { luke.email }
+    )
+  end
 
-    expect(1).to eq(1)
+  example "Skipping some records when a symbol is passed to :unless option" do
+    User.class_eval do
+      attr_masker :first_name, :last_name, unless: :jedi?
+    end
+
+    expect { run_rake_task }.not_to(change { User.count })
+
+    expect { han.reload }.to(
+      change { han.first_name }.to("(redacted)") &
+      change { han.last_name }.to("(redacted)") &
+      preserve { han.email }
+    )
+
+    expect { luke.reload }.to(
+      preserve { luke.first_name } &
+      preserve { luke.last_name } &
+      preserve { luke.email }
+    )
+  end
+
+  example "Skipping some records when a lambda is passed to :unless option" do
+    User.class_eval do
+      attr_masker :first_name, :last_name, unless: ->(r){ r.jedi? }
+    end
+
+    expect { run_rake_task }.not_to(change { User.count })
+
+    expect { han.reload }.to(
+      change { han.first_name }.to("(redacted)") &
+      change { han.last_name }.to("(redacted)") &
+      preserve { han.email }
+    )
+
+    expect { luke.reload }.to(
+      preserve { luke.first_name } &
+      preserve { luke.last_name } &
+      preserve { luke.email }
```

Comment:
@ronaldtse @abunashir Similar thing here. Really no point in placing a comma here, IMO.